code/modules/food/recipes_microwave.dm

```diff
@@ -951,22 +951,20 @@ I said no!
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/tortilla,
 		/obj/item/weapon/reagent_containers/food/snacks/meatball,
-		/obj/item/weapon/reagent_containers/food/snacks/meatball,
 		/obj/item/weapon/reagent_containers/food/snacks/meatball
 	)
 	reagents = list("spacespice" = 1)
 	result = /obj/item/weapon/reagent_containers/food/snacks/burrito
 
 /datum/recipe/burrito_vegan
```

Comment:
This is the most depressing excuse of a burrito I've ever seen conceived. where's the flavor? where's the heart? where's the _soul_? at this point you might as well just find a sheet of paper and bite into it.

Seriously, this is insulting to anyone who might be tricked into thinking this is *food*, and insulting to burritos as a food group.

A burrito by definition requires a filling, but calling tofu filling is a joke. you might as well be saying that Security is competent, or that I have any idea what I'm doing.
Sure it's _supposedly_ true, but stating it makes you look like a massive dingus simply because everyone knows it's false. a burrito is a god damn art form and tarnishing it by making it _only_ tofu should be a sin. I have never in my life heard of such a pathetic and tasteless food before this, and that's simply not OK. see some help.

Reply:
don't @ me